config.py

```diff
@@ -1,16 +1,16 @@
 import os
 user = "root"
-password = "root"
+password = "Ritikjain@1272"
 host = "localhost"
 database = "mapbot"
 port = "3306"
-key = "*Google_Cloud_API_key*"  # Will be provided by mentors
+key = "AIzaSyBuHPXeUJKVTiHwjY8m9bw4Rvv1fYOCxmk"  # Will be provided by mentors
```

Comment:
While sharing the API key, the instructions were clear that it must *never* be shared. Why has the API key been shared in a commit? Even further, why is `config.py` even added to the commit?